README.md

```diff
@@ -21,3 +21,4 @@ PowerShell
 SharePoint
 Mobile Device Management with WorkSpace One (formerly AirWatch) and InTune
 
+
```

Comment:
Because you added the `answer-one` sections to your `answer-two` branch your conflict didn't present the way it should. You got a conflict and resolved but be careful to follow the steps.